app/converters/shopify/variant_converter.rb

```diff
@@ -0,0 +1,35 @@
+module Shopify
+  class VariantConverter
+    def initialize(spree_variant)
+      @spree_variant = spree_variant
+    end
+
+    def to_hash
+      {
+        product_id: spree_variant.product_id,
+        weight: spree_variant.weight,
+        weight_unit: spree_variant.weight_unit,
+        price: spree_variant.price,
+        sku: spree_variant.sku,
+        updated_at: spree_variant.updated_at
+      }.merge(options).merge(variant_uniqueness_constraint)
+    end
+
+    private
+
+    attr_reader :spree_variant
+
+    def options
+      options = {}
```

Comment:
Would this be better implemented during the initialization process and then given a private reader method?